### PR TITLE
Added Pack to raw_ops

### PR DESCRIPTION
**Pull request by KareemMAX**
Close https://github.com/unifyai/ivy/issues/5407

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py [hunk 1]
```diff
@@ -2231,3 +2231,31 @@ def test_tensorflow_Unpack(
         num=x[0].shape[axis],
         axis=axis,
     )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid", full=True),
```
Comment (CatB1t) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:2239]: No need to specify `full=True` here, it's `True` by default.

**Review by CatB1t — CHANGES_REQUESTED** [review]
just a small change, otherwise lgtm!

**Comment by KareemMAX** [comment]
Fixed the issue and resolved the conflicts I had

**Comment by CatB1t** [comment]
lgtm! could you resolve the conflict? then we're good to merge 😄

**Review by CatB1t — APPROVED** [review]

**Review by CatB1t — APPROVED** [review]

**Comment by KareemMAX** [comment]
@CatB1t Conflicts resolved

**Pull request merged**